0miner

```diff
@@ -462,37 +375,20 @@ then
   screen -dmSL miner $HCD -o stratum+tcp://$SUMO_POOL:$SUMO_PORT -u $ADDR -p $MINER_PWD $CCMINER_OPTS
 fi
 
-if [[ $COIN == NICE-NEOSCRYPT ]]
-then
-  HCD=${NVOC}/miners/"$NEOSCRYPT_MINER"/ccminer
-  ADDR="$NICE_ADDRESS$WALLET_ADDRESS_FORMAT$NICE_WORKER"
-  screen -c ${NVOC}/screenrc-miner -dmSL miner $HCD -a neoscrypt -o stratum+tcp://$NICE_NEOSCRYPT_POOL:$NICE_NEOSCRYPT_PORT -u $ADDR -p x -i $NEOSCRYPT_INTENSITY $CCMINER_OPTS
-fi
-
-if [[ $COIN == NICE-LYRA2REV2 ]]
-then
-  HCD=${NVOC}/miners/"$LYRA2V2_MINER"/ccminer
-  ADDR="$NICE_ADDRESS$WALLET_ADDRESS_FORMAT$NICE_WORKER"
-  screen -c ${NVOC}/screenrc-miner -dmSL miner $HCD -a lyra2v2 -o stratum+tcp://$NICE_LYRA2REV2_POOL:$NICE_LYRA2REV2_PORT -u $ADDR -p x -i $LYRA2V2_INTENSITY $CCMINER_OPTS
-fi
-
 if [[ $COIN == NICE-X11GOST ]]
 then
   HCD=${NVOC}/miners/SPccminer/ccminer
-  ADDR="$$NICE_ADDRESS$WALLET_ADDRESS_FORMAT$NICE_WORKER"
-  screen -c ${NVOC}/screenrc-miner -dmSL miner $HCD -a sib -o stratum+tcp://$NICE_X11GOST_POOL:$NICE_X11GOST_PORT -u $ADDR -p x -i $NICE_X11GOST_INTENSITY $CCMINER_OPTS
+  screen -c ${NVOC}/screenrc-miner -dmSL miner $HCD -a sib -o stratum+tcp://$NICE_X11GOST_POOL:$NICE_X11GOST_PORT -u $NICE_ADDR -p x -i $NICE_X11GOST_INTENSITY $CCMINER_OPTS
 fi
 
 if [[ $COIN == NICE-SKUNKHASH ]]
 then
   HCD=${NVOC}/miners/"$SKUNK_MINER"/ccminer
```

Comment:
Why not `HCD="${NVOC}/miners/$SKUNK_MINER/ccminer"` ? Why quoting this one and a few other only?

Reply:
Will run shellcheck on it when all done.